src/action/tests/SummationActionTest.cpp

```diff
@@ -0,0 +1,200 @@
+#include "action/SummationAction.hpp"
+#include "action/config/ActionConfiguration.hpp"
+#include "mesh/Data.hpp"
+#include "mesh/Edge.hpp"
+#include "mesh/Mesh.hpp"
+#include "mesh/Vertex.hpp"
+#include "mesh/config/DataConfiguration.hpp"
+#include "mesh/config/MeshConfiguration.hpp"
+#include "testing/Testing.hpp"
+
+using namespace precice;
+
+BOOST_AUTO_TEST_SUITE(ActionTests)
+BOOST_AUTO_TEST_SUITE(Summation, *testing::OnMaster())
+
+BOOST_AUTO_TEST_CASE(SummationTwoSources)
+{
+  using namespace mesh;
+  PtrMesh mesh(new Mesh("Mesh", 3, true, testing::nextMeshID()));
+  PtrData sourceData1   = mesh->createData("SourceData1", 1);
```

Comment:
```suggestion
  int dimension = 1;
  PtrData sourceData1   = mesh->createData("SourceData1", dimension);
```
Makes it easier to read.